pkgs/by-name/ce/cert-viewer/package.nix

```diff
@@ -21,10 +23,17 @@ buildGoModule rec {
     installShellFiles
   ];
 
-  postInstall = ''
-    $out/bin/cert-viewer --help-man > cert-viewer.1
-    installManPage cert-viewer.1
-  '';
+  postInstall =
+    let
+      prog =
+        if stdenv.buildPlatform.canExecute stdenv.hostPlatform
+        then "$out/bin/cert-viewer"
+        else lib.getExe buildPackages.cert-viewer;
```

Comment:
It can always be used?

Reply:
You mean buildPackages.cert-viewer? I did that at first, but hen realised it creates infinite recursion.

Reply:
Oh got it...